admin/controllers/content/views/order/view.php

```diff
@@ -0,0 +1,151 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+// TODO: handle custom states - see all view
+// TODO: handle base grid column count - see all view
+
+?>
+
+<style>
+    <?php echo file_get_contents(CMSPATH . "/admin/controllers/content/views/all/style.css"); ?>
+    #orderitems {}
+    .orderitem {
+        /* display:flex;
+        gap:1rem; */
+        &:hover {
+            cursor:move;
+        }
+    }
+    .sortable-ghost {
+    }
+    .sortable-chosen {
+        font-weight:bold;
+    }
+    .sortable-drag {
+    }
+    table#orderitems tr {
+        grid-template-columns: repeat(<?php echo sizeof($content_list_fields)+4;?>, 1fr); /* match # of columns in ordering table */
+    }
+    table.table td:nth-of-type(<?php echo Admin_Config::$show_ids_in_tables ? 3 : 2; ?>), table.table th:nth-of-type(<?php echo Admin_Config::$show_ids_in_tables ? 3 : 2; ?>) {
+        grid-column: span 3;
+    }
```

Comment:
copy pasted code that isnt used?

Reply:
it's used - copied but tweaked for this view